sbt-plugin-test/build.sbt

```diff
@@ -135,6 +135,12 @@ lazy val jsDependenciesTest = project.settings(versionSettings: _*).
   )): _*).
   dependsOn(jetty9) // depends on jQuery
 
+lazy val issue1571 = project.
+  enablePlugins(ScalaJSPlugin).
+  settings(versionSettings: _*).
+  settings(jsDependencies += ProvidedJS / "consoleWriter.js").
+  dependsOn(testFrameworkJS % "test")
+
```

Comment:
I don't think we need a separate project for this. You can just add `consoleWriter.js` to the `jsSettings` of `multiTest` with a small comment saying what this tests and reference the issue (we usually write "- #xyz").